QSB/TransformSync/TransformSync.cs

```diff
@@ -56,12 +58,12 @@ protected void Reset()
         private void SetFirstSector()
         {
             _isSectorSetUp = true;
-            PlayerRegistry.LocalPlayer.ReferenceSector = Locator.GetAstroObject(AstroObject.Name.TimberHearth).transform;
+            ReferenceTransform = Locator.GetAstroObject(AstroObject.Name.TimberHearth).transform;
         }
 
         public void EnterSector(Sector sector)
         {
-            SectorSync.Instance.SetSector(PlayerRegistry.LocalPlayer.NetId, sector.GetName());
+            SectorSync.Instance.SetSector(netId.Value, sector.GetName());
```

Comment:
But every TransformSync instance has a different netId... how do we know which ones applies to which players? or am I missing something

Reply:
We don't, and don't need to, for this. TransformSyncs and players are separate things.

They could be tied together but that's another refactoring

Reply:
ReferenceSector was a part of PlayerInfo but that was part of the issue introduced in nebula-development where only local player's sector was synces (and no other players and no ships). check the diff of nebula-development for clarification. there you'll se how things were - all transformsyncs were synced, independently of players